modules/programs/irssi.nix

```diff
@@ -23,6 +23,12 @@ let
         type = "${v.type}";
         nick = "${quoteStr v.nick}";
         autosendcmd = "${concatMapStringsSep ";" quoteStr v.autoCommands}";
+        ${
+          lib.optionalString (v.saslExternal) ''
+            sasl_username = "${quoteStr v.nick}";
+              sasl_mechanism = "EXTERNAL";
```

Comment:
I had to add two extra spaces on line 29 to get consistent indentation in the config produced in the test. I'm not sure if that's the correct thing to do though.